### PR TITLE
Update to allow version check to work with php 8

### PR DESCRIPTION
**Pull request by timhaak**
Trying to start getting things to work with PHP 8.0

Also fixed the PHPUnit deprecation warning.

### CODE REVIEW

**Comment by soullivaneuh** [comment]
@timhaak Thanks for this contribution! The CI is failing, could you please take a look?

**Comment by soullivaneuh** [comment]
Is this PR whiling to resolve https://github.com/soullivaneuh/composer-versions-check/issues/74?

**Comment by timhaak** [comment]
Files are on the ftp server here 

/uploads/thumbnails/

I’ve tried to see any that were blank and replace with an image later in the video.

Tim Haak
+27 83 778 7100
tim@haak.co <mailto:tim@haak.co>

**Comment by timhaak** [comment]
Just a bit to busy to sort this out now.

Will do a new pull request with fixes when/if I catch up a bit :)

File: src/VersionsCheck.php [hunk 1]
```diff
@@ -50,11 +49,24 @@ public function checkPackages(ArrayRepository $distRepository, WritableRepositor
...
                 // Sort packages by highest version to lowest
                 usort($higherPackages, function (PackageInterface $p1, PackageInterface $p2) {
-                    return Comparator::compare($p1->getVersion(), '<', $p2->getVersion());
+                    if (Comparator::compare($p1->getVersion(), '=', $p2->getVersion())) {
+                        return 0;
+                    }
+                    if (Comparator::compare($p1->getVersion(), '<', $p2->getVersion())) {
+                        return 1;
+                    }
+                    return -1;
```
Comment (pelmered) [inline src/VersionsCheck.php:58]: Replace with ternary:
 `return Comparator::compare($p1->getVersion(), '<', $p2->getVersion()) ? 1 : -1;`
There shouldn't be any versions that are equal, right?

**Pull request closed without merging**